af/analysis/rotation.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+"""
+Spaws and rotates hosts on Digital Ocean. Runs a setup script on the host
+at first boot.
+Reads /etc/ooni/rotation.conf
+Keeps a list of live hosts in a dedicated db table.
+Runs as a SystedD timer.
+
+Table setup:
+CREATE UNLOGGED TABLE test_helper_instances (
+    name text NOT NULL,
+    provider text NOT NULL,
+    region text,
+    ipaddr inet NOT NULL,
+    ipv6addr inet,
+    draining_at timestamp without time zone
+);
+ALTER TABLE test_helper_instances OWNER TO shovel;
+
+Example for /etc/ooni/rotation.conf
+--
+[DEFAULT]
+token = CHANGEME
+db_uri = postgresql://shovel:CHANGEME@localhost/metadb
+active_droplets_count = 4
```

Comment:
Shouldn't there be in the settings also `draining_time_minutes`?

Reply:
Added